README.md

```diff
@@ -109,19 +109,17 @@ Edit the `.env` file with the necessary settings.
 # Replace the credentials here with your own.
 # Rename this file to .env before starting the app.
 
-# Watson Services
-# IMPORTANT: If your service is provisioned with IAM authentication, comment out the
-# username/password credentials and use the IAM credentials key/value pair
-
-# Watson Discovery authentication using username/password authentication
-DISCOVERY_USERNAME = <put discovery username here>
-DISCOVERY_PASSWORD = <put discovery password here>
-# Watson Discovery authentication using IAM
-#DISCOVERY_IAM_APIKEY=<put discovery IAM apikey here>
-#DISCOVERY_URL=<put discovery url here>
-
-DISCOVERY_ENVIRONMENT_ID=<put discovery environment id here>
-DISCOVERY_COLLECTION_ID=<put discovery collection id here>
+# Replace the credentials here with your own.
+# Rename this file to .env before starting the app.
```

Comment:
unrelated, but I happen to think it is better to "Copy" and then edit instead of mucking w/ a file that'll show up as a repo change